wcmc_components/app/controllers/wcmc_components/table_controller.rb

```diff
@@ -48,7 +49,7 @@ def update
 
       if @table_resource.save
         # Redirect (to the show page/index page)
-        render :edit
```

Comment:
also needs to be added to new and edit actions, and for now including in the create links in the IR pages (should eventually be a button in the front end library)